scripts/buildtools/go.sum

```diff
@@ -23,6 +23,8 @@ github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuy
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/algorand/msgp v1.1.50 h1:Mvsjs5LCE6HsXXbwJXD8ol1Y+c+QMoFNM4j0CY+mFGo=
```

Comment:
go mod tidy to remove the old version?

Reply:
I tried and it cleared out all of go.mod! I guess all of these packages are unused. TBH I'm not really sure long-term whether it is a good idea to keep installing CLI tools this way.

Reply:
I see now that install_buildtools.sh is just grepping go.mod to pull version numbers, so I changed it to a flat text file.

Reply:
Does that make it harder to create though? Presumably it was created my some go tool automatically, but the `versions` file will need to be edited by hand.  Could it grep the root go.mod instead? I don't really know what this one is, how it was made, or why it might differ.

Reply:
IMO it is much easier to maintain a simple "versions" file now by hand — I would even be OK to just move the tool versions into install_buildtools.sh directly, as it's the only user. The reason it's not in root go.mod is because these tools are not dependencies of go-algorand. (They don't even really need to be implemented in go — they're just CLI tools to install)